### PR TITLE
add read/liveness probes to the stateful set and test case

### PR DESCRIPTION
**Pull request by cpanato**
**What this PR does / why we need it**:
follow up of https://github.com/helm/charts/pull/7588#pullrequestreview-153203241

**Which issue this PR fixes** : fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/minio/templates/statefulset.yaml [hunk 1]
```diff
@@ -62,6 +62,15 @@ spec:
                 secretKeyRef:
                   name: {{ template "minio.fullname" . }}
                   key: secretkey
+          livenessProbe:
+            tcpSocket:
+              port: service
+            initialDelaySeconds: 5
```
Comment (davidkarlsen) [inline stable/minio/templates/statefulset.yaml]: would it make sense to make initialDelaySeconds and timeoutSeconds configurable (and how many retries) - or are these bound to always be fast?

Comment (cpanato) [reply stable/minio/templates/statefulset.yaml]: @davidkarlsen that makes sense to have configurable with these default values and if the user wants to change will be easier. I will add that

thanks for the review

Comment (cpanato) [reply stable/minio/templates/statefulset.yaml]: @davidkarlsen PTAL

File: stable/minio/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Minio is a high performance distributed object storage server, designed for large-scale private cloud infrastructure.
 name: minio
-version: 1.6.5
+version: 1.6.6
```
Comment (davidkarlsen) [inline stable/minio/Chart.yaml]: new features 1.7.0? I think this one is ready to go now - great PR

Comment (cpanato) [reply stable/minio/Chart.yaml]: cool, thanks! PTAL

**Pull request merged**